src/containers/TemplateBuilder/shared/OperationContextHelpers.tsx

```diff
@@ -270,14 +277,15 @@ export const importTemplate: ImportTemplateHelper =
     }
   }
 
-const getFitlerBody = (id: number) => {
+const getFilterBody = (id: number) => {
   const equalToTemplateId = { equalTo: id }
   const allElementsMatchTemplateId = { some: { templateId: equalToTemplateId } }
   const filters = {
     filters: {
       template: { id: equalToTemplateId },
       filter: { templateFilterJoins: allElementsMatchTemplateId },
       permissionName: { templatePermissions: allElementsMatchTemplateId },
+      templateCategory: { templates: { some: { id: { equalTo: id } } } },
```

Comment:
This is the only actual change -- just limits the categories exported to ones linked to the template.